web/src/components/MemoEditor.tsx

```diff
@@ -121,6 +121,15 @@ const MemoEditor = () => {
         editorRef.current.insertText("", "`", "`");
         return;
       }
+      if (event.key === "k") {
+        event.preventDefault();
+        const selectedContent = editorRef.current.getSelectedContent();
+        editorRef.current.insertText("", "[", "]()");
```

Comment:
```suggestion
        editorRef.current.insertText("", "[", "](url)");
```
<img width="260" alt="image" src="https://user-images.githubusercontent.com/24653555/208636372-e5239349-9493-4b96-8472-de3d5de49bc7.png">

Reply:
When selecting a part of the content should be `[select range text](|)`?

Reply:
Yeah, I think it should be `[selected text](url)`. You can checkout the behavir in this GitHub editor.